src/sync.ts

```diff
@@ -94,16 +89,25 @@ export async function exec(cli: meow.Result) {
   }
 
   logger.info(`Executing '${command}' in ${dirs.length} directories.`);
-  for (let i = 0; i < dirs.length; i++) {
-    const dir = dirs[i];
-    logger.info(`[${i + 1}/${dirs.length}] Executing cmd in ${dir}...`);
-    try {
-      await spawn(command[0], command.slice(1), {cwd: dir});
-    } catch (e) {
-      logger.error(dir);
-      logger.error(e);
-    }
-  }
+  let i = 0;
+  const q = new Q({concurrency: 10});
```

Comment:
When I see integer constants like `10` in the code, I always ask to make it configurable :)

Reply:
I don't think there's a lot of value in making this end user configurable.  I'd be happy to entertain changing it tho :)